src/Data/Object/Simulator.elm

```diff
@@ -58,24 +69,56 @@ compute db query =
 
 
 computeItemResults : Db -> Item -> Result String Results
-computeItemResults { object } { amount, processId } =
+computeItemResults db item =
+    item.processes
+        |> List.map (computeProcessItemResults db)
+        |> RE.combine
+        |> Result.map
+            (List.foldr
+                (\(Results results) (Results acc) ->
+                    Results
+                        { acc
+                            | impacts = Impact.sumImpacts [ results.impacts, acc.impacts ]
+                            , items = Results results :: acc.items
+                            , mass = Quantity.sum [ results.mass, acc.mass ]
+                        }
+                )
```

Comment:
From a distance, it feels like this lambda could be factored out and used in both `compute` and `computeItemResults`